### PR TITLE
[DC 2021] Launch with dates

### PR DESCRIPTION
**Pull request by nathenharvey**
Signed-off-by: Nathen Harvey <nathen.harvey@gmail.com>

### CODE REVIEW

**Review by robbkidd — APPROVED** [review]
Announcing ...

**Review by MikeRosTX — CHANGES_REQUESTED** [review]
Pleas set all 3 CFP date following the same format syntax as start and end dates.

**set all three at once**

File: data/events/2021-washington-dc.yml [hunk 1]
```diff
@@ -10,11 +10,11 @@ ga_tracking_id: "" # If you have your own Google Analytics tracking ID, enter it
...
 # Note: we allow 2021-MM-DD for backward compatibility, but it can lead to unexpected behaviors (like your event disappearing from the front page during your last day)
 
-startdate:  # The start date of your event. Leave blank if you don't have a venue reserved yet.
-enddate:  # The end date of your event. Leave blank if you don't have a venue reserved yet.
+startdate:  2021-09-16T08:00:00-04:00
+enddate:  2021-09-17T19:00:00-04:00
 
 # Leave CFP dates blank if you don't know yet, or set all three at once.
-cfp_date_start:  # start accepting talk proposals.
+cfp_date_start:  2021-04-16
```
Comment (robbkidd) [inline data/events/2021-washington-dc.yml:17]:
```suggestion
cfp_date_start:  2021-04-16T00:00:00-04:00
```

File: data/events/2021-washington-dc.yml [hunk 2]
```diff
@@ -10,11 +10,11 @@ ga_tracking_id: "" # If you have your own Google Analytics tracking ID, enter it
...
 cfp_date_end:  # close your call for proposals.
 cfp_date_announce:  # inform proposers of status
```
Comment (robbkidd) [inline data/events/2021-washington-dc.yml:19]: @nathenharvey Do we have the timeline for the rest of the CFP process (closure and accept/decline announcement)?

**Review by MikeRosTX — APPROVED** [review]

**Review by MikeRosTX — APPROVED** [review]
Looks fine by me. 😉 👍

**Comment by MikeRosTX** [comment]
@nathenharvey feel free to merge if you like, or I can do it for you.

**Pull request merged**